### ISSUE TITLE
Track information

### ISSUE THREAD

**Issue by darkzbaron** [vishen/go-chromecast#95]
Hi there, is there a way to populate currently playing artist, title etc? The fields are there but when using the load function (both with server and bin) it does not work.

Thanks very much in advance.

Kind regards, Jonathan

**Comment by vishen** [comment]
Not at the moment, we don't currently try and get any metadata about the file we have loaded. 

It is possible to set the playing artist, title etc when loading custom media, but I am not too sure how you would go about getting the metadata from the file you want to play. I suspect some media formats include the metadata in the file? So this would likely need to be done on a media format basis.

**Comment by darkzbaron** [comment]
Yes it is a mess, I have used Mutagen to get track information for most fortmat I use. How would you go about to send the info to the Chromecast device?

**Comment by vishen** [comment]
Yeah, if it was to be implemented it would only work for custom media that is loaded. Usually each chromecast app (Netflix, Plex, etc) will add the track information if it's available.
